src/DIRAC/WorkloadManagementSystem/ConfigTemplate.cfg

```diff
@@ -283,6 +283,12 @@ Agents
     SendPilotSubmissionAccounting = True
   }
   ##END
+  ##BEGIN PushJobAgent
+  PushJobAgent
+  {
+    MaxJobsToSubmit = 100
```

Comment:
Please add a comment, it would appear in the CS once installed.

Also, add this to PushJobAgent docstring:

```
.. literalinclude:: ../ConfigTemplate.cfg
  :start-after: ##BEGIN PushJobAgent
  :end-before: ##END
  :dedent: 2
  :caption: PushJobAgent options
```